.gitignore

```diff
@@ -7,3 +7,6 @@ node_modules
 terraform.tfstate*
 terraform.tfvars
 .terraform
+
+# macos cruft
+.DS_Store
```

Comment:
This should be set in one's global gitignore.

Reply:
Good point.  I'll remove from here.
@christopher-maboh for visibility.

Reply:
Packer already takes AMI as [user param](https://github.com/CMSgov/bcda-ops/blob/master/packer/jenkins.json#L14), so as long as it is passed the AMI ID ami-076e72264a6f55452 or other encrypted AMI base, it. should work without changes.  I'll try to find appropriate documentation and update to reflect this.

Reply:
@christopher-maboh These lines can still be removed.